pombola/writeinpublic/views.py

```diff
@@ -26,14 +27,15 @@ def wrap(request, *args, **kwargs):
 
 
 class WriteInPublicMixin(object):
-    def __init__(self, *args, **kwargs):
+    def get(self, *args, **kwargs):
```

Comment:
This should be overriding the `dispatch` method, not the `get` method, otherwise it won't work with form POSTs.

Reply:
It turns out that `self.client` is never called from a POST request. When the POST is done to the "preview" step the wizard then redirects to the "done" step, so the browser does a GET request to that step, which is where `self.client` is used.

I've decided to change this to `dispatch` _anyway_, because it seems like a more robust approach for the future.